src/ApplicationInsights.Kubernetes/ContainerIdProviders/NullContainerIdProvider.cs

```diff
@@ -0,0 +1,18 @@
+﻿#nullable enable
+
+namespace Microsoft.ApplicationInsights.Kubernetes.ContainerIdProviders
+{
+    internal class EmptyContainerIdProvider : IContainerIdProvider
```

Comment:
nit: class name does not correspond to file name

Reply:
> Consider naming the class `WindowsContainerIdProvider`. The fact that it is currently unable to retrieve a container ID may change in future.

Thanks for notice the mismatch.

I renamed the file to `EmptyContainerIdProvider`. If/When there's new way to retrieve the container id for windows container, that would be the new class of `WindowsContaienrIdProvider` being appended.

Currently, the name of `EmptyContainerIdProvider` reveals what the class does.